<dec f='qtbase-6.5.0/src/network/access/qnetworkreply.h' l='71' type='401'/>
<doc f='qtbase-6.5.0/src/network/access/qnetworkreply.h' l='70'>// Server side errors (401-499)</doc>
<use f='qtbase-6.5.0/src/network/access/http2/http2protocol.cpp' l='116' u='r' c='_ZN5Http28qt_errorEjRN13QNetworkReply12NetworkErrorER7QString'/>
<use f='qtbase-6.5.0/src/network/access/qhttpthreaddelegate.cpp' l='63' u='r' c='_ZL18statusCodeFromHttpiRK4QUrl'/>
<doc f='qtbase-6.5.0/src/network/access/qnetworkreply.cpp' l='159'>\value InternalServerError          the server encountered an unexpected
    condition which prevented it from fulfilling the request.</doc>
